WickedEngine/wiSpriteAnim_BindLua.cpp

```diff
@@ -342,24 +342,23 @@ namespace wi::lua
 		lunamethod(DrawRectAnim_BindLua, GetFrameRate),
 		lunamethod(DrawRectAnim_BindLua, GetFrameCount),
 		lunamethod(DrawRectAnim_BindLua, GetHorizontalFrameCount),
-		{ NULL, NULL }
+		{ nullptr, nullptr }
 	};
 	Luna<DrawRectAnim_BindLua>::PropertyType DrawRectAnim_BindLua::properties[] = {
-		{ NULL, NULL }
+		{ nullptr, nullptr }
 	};
 
-	DrawRectAnim_BindLua::DrawRectAnim_BindLua(const wi::Sprite::Anim::DrawRectAnim& data) :anim(anim)
-	{
-	}
+	DrawRectAnim_BindLua::DrawRectAnim_BindLua(const wi::Sprite::Anim::DrawRectAnim& data)
+		:anim(data)
```

Comment:
This one is a nice find

Reply:
This one specifically I will fix separately.

Reply:
I'm thinking of enabling this particular warning on Linux as an error. It seems pretty useful and we automatically test it in the CI.

Reply:
ok, if this can be enabled that's useful